site2/website/release-notes.md

```diff
@@ -1,11 +1,98 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.7.2 &mdash; 2021-04-XX <a id=“2.7.2”></a>
```

Comment:
What's the detailed date for it? You can specify it here.

Reply:
I will amend the PR once the VOTE is closed.

When the release is out I will update the PR and then we can merge it.

Reply:
thank you.